README.md

```diff
@@ -31,3 +31,7 @@ Performs the equivalent of a `git coco create` followed by `git coco clone`. The
 * `git coco pr <from> <to>`
 
 Creates a pull request for the current repository that compares `<from>` (or the current branch if omitted) with `<to>` (or `master` if omitted).
+
+* `git coco tag <repo> <tagKey=tagValue,tagKey2=tagValue2,...>`
+
+Adds tags to CodeCommit repository
```

Comment:
It might be useful to note why one might want to do that.  Does it remove existing tags and replace them with the specified set, or add to them ?

Reply:
Good idea, I have added more description with your recommendations.